ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+# Who uses GitOps? 
+
+As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
+
+Currently, the following organizations are officially adopting GitOps (in alphabetical order). 
+
+1. [Example Organization](https://example.com)
```

Comment:
What about including a simple example for adopters to describe how they're adopting GitOps?

```suggestion
1. [Example Organization](https://example.com) – `Example` has adopted GitOps to manage our X and Y products and their required infrastructure. We use A and B tools/techniques, and participate in the GitOps WG meetings. See this a (blog post/talk) about the details of our journey and current setup: <https://example.com/blog/how-we-do-gitops>
```